src/core/sdl/render_sdl.cpp

```diff
@@ -18,6 +18,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <SDL2/SDL.h>
```

Comment:
What the reason to include the main SDL header? Does it contain anything that other smaler headers don't have? (for example, `SDL_Init()` or `SDL_Quit()`). But, I guess, `SDL_Hint` also here, but I don't remind exactly...

Reply:
Ah, SDL_hint, thank you.